pkg/apis/networking/v1alpha1/ingress_validation.go

```diff
@@ -107,7 +109,10 @@ func (h HTTPIngressPath) Validate(ctx context.Context) *apis.FieldError {
 				Paths:   []string{"splits"},
 			})
 		}
+	case h.RewriteHost != "":
```

Comment:
Can we remove this case if no validation is needed?

Reply:
I think it's marginally clearer with all the cases of the switch exhaustively laid out for the reader but.. certainly not enough to bikeshed about :) Removed.